src/CMakeLists.txt

```diff
@@ -224,7 +222,7 @@ add_subdirectory(cppcache)
 add_subdirectory(cryptoimpl)
 add_subdirectory(dhimpl)
 add_subdirectory(sqliteimpl)
-add_subdirectory(gfcpp)
+add_subdirectory(getversion)
```

Comment:
If they are relying on it then their scripts will break either way. There are better ways to get version. I don't think I have ever seen anyone anywhere use it. I don't even see it in our documents. I say kill it!